portfolio/src/main/webapp/index.html

```diff
@@ -1,65 +1,102 @@
+                <h4 id="quote-container" style="color:#555;"></h4>
             </div>
-            <br>
-            <div id="terminal" class="col-md-5">
-                <p><br>Welcome to My website!<br>Get to know me better! Type "help" to get started. </p>
-				<span>$</span> <input type='text' id = "choice"  autofocus>
-                <p id="output"></p>
-            </div>           
-           
-        </div>
-    </div>
-
-    
-</body>
-<script src="script.js"></script>
+            <div class="container-fluid">
+               <div class="row">
+                  <div class="col-md-1"></div>
+                  <div id="picture" class="col-md-3 ">
+                     <img id="profile-pic" src="assets/images/profile.jpeg" alt="That's me!">
+                  </div>
+                  <div class="col-md-1"></div>
+                  <div class="col-md-6" id ="intro">
+                     <br>
+                     <h3><b>Hello World!</b> </h3>
+                     <br />
+                     <h5> I’m Naveena, a 20-year-old Computer Science Engineering student at NIT-Trichy. I love exploring different fields of Computer Science and learning how it can affect our lives in different ways. 
+                        Some parts of the computer sciences that interest me the most are server-side development, cryptography and blockchain technology.<br>
+                     </h5>
+                     <br>
+                     <h4>Checkout my online profiles here :</h4>
+                     <br>
+                     <p>
+                        <a href="assets/resume.pdf" target="_blank"
+                           ><img src="https://img.icons8.com/windows/48/000000/resume.png" /></a>
+                        <a href="https://linkedin.com/in/na899" target="_blank"
+                           >
+                        <img src="https://img.icons8.com/windows/48/000000/linkedin.png" /></a>
+                        <a href="https://github.com/na899" target="_blank">
+                        <img src="https://img.icons8.com/windows/48/000000/github.png" /></a>
+                        <a href="https://medium.com/@pnaveena1999" target="_blank"> 
+                        <img src="https://img.icons8.com/windows/48/000000/medium-logo.png" /></a>
+                     </p>
+                  </div>
+               </div>
+               <div class="row">
+                  <div class="col-md-4"></div>
+                  <div class="col-md-4"> <img src="https://img.icons8.com/ios-filled/50/000000/down.png"/> <br>
+                     Keep scrolling to explore!
+                  </div>
+                  <div class="col-md-4"></div>
+               </div>
+            </div>
+         </section>
+         <section class="even  align-items-centers" id="two">
+            <h1>Terminal</h1>
+            <div id="terminal" >
+               Welcome to My website! Get to know me better! Type "help" to get started. <br>

```

Comment:
I am unable to type anything
Is that expected?

Reply:
The user should click on the input before typing. Tried to autofocus based on scroll position, but i read that it's a bad practice since we are taking control from the user for scrolling. So now, i've added a note to ask the user.